{{cookiecutter.project_slug}}/run-server.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-PORT=${PORT:-8000}
```

Comment:
keep the file or move it to makefile. It's more about having it self documented how to run the server. 
could also be with the cli.